qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -270,32 +270,46 @@ class TwoQubitControlledUDecomposer:
     :math:`U \sim U_d(\alpha, 0, 0) \sim \text{Ctrl-U}`
     gate that is locally equivalent to an :class:`.RXXGate`."""
 
-    def __init__(self, rxx_equivalent_gate: Type[Gate]):
+    def __init__(self, rxx_equivalent_gate: Type[Gate], euler_basis: str = "ZYZ"):
         r"""Initialize the KAK decomposition.
```

Comment:
should we replace the default value to "ZXZ" to match the current fractional gates `rx` and `rz` ?
(If so, we should do this more generally in other places of the code in this file)

Reply:
I replaced "ZYZ" to "ZXZ". Should we also do it in other classes ?

Reply:
Mmm I don't really know what motivated the original choice of default gates, maybe it was done on purpose not to match the current fractional gates? @mtreinish do you have any insight?

Reply:
I would avoid changing the defaults for any other classes it causes necessary churn. I think the choice was more just out of habit/standard gates to use. In this case using zxz as the default is fine since it's new, (although I'd be inclined to use `zsxx` as the default). But it doesn't really matter because either you don't care about the euler basis, or you do and you will specify it.

Reply:
I think that with RZZ gates the one-qubit gates are RZ and RX, so it makes sense to choose the default ZXZ here: https://docs.quantum.ibm.com/guides/fractional-gates
For other 2-qubit gates (CX/CZ/ECR) the one-qubit gates are still ZX, SX and X so it makes sense to choose other defaults. So I agree that we don't need to change default values in other parts of the code.